### PR TITLE
React Calculator - Milestone #4 - Calculations

### PR DESCRIPTION
**Pull request by mateomh**
In this milestone, 2 modules were created to help with the logic of the calculator:

- The **calculate** module is supposed to keep track of all the keys pressed in the calculator.
- The **operate** module is in charge of making the calculations for the selected operation.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  let result = new Big(0);
+  const number1 = new Big(numberOne);
+  const number2 = new Big(numberTwo);
+  switch (operation) {
```
Comment (wathigo) [inline src/logic/operate.js:7]: Let's include support for modifier operator(%), which turns the number into a percentage based number, e.g. `95 % == .95`. [Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png).

**Review by wathigo — CHANGES_REQUESTED** [review]
### Changes Required :heavy_multiplication_x: 
Great job on this milestone team.
Let's fix the issue pointed out in the comment :point_down:  before we move on.

**After making the changes, submit another request for review.**

**Happy Coding :technologist: 
Simon Wathigo.**

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @mateomh :hand: 
Great job so far in this milestone. And also, a great job implementing the requested changes :clap: :+1: 
However there it's one change that we need to make before moving on.

- [x] The linter.yml file it's not the same one as the one provided by Microverse. We need to use the most recent one, you can find it [here](https://github.com/microverseinc/linters-config/blob/master/react-redux/.github/workflows/linters.yml).

Feel free to message me on slack if you have any questions :smile: 
Happy Coding :computer: 
**Sergio Torres**

**Comment by mateomh** [comment]
Thanks Sergio for the review. I tried with the provided one but with the import plugin 2.20 for estlint it doesn't work nor in the local environment or the GitHub actions, it throws the following error:

image

With the 2.22 version, it goes away. This error didn't happen with JS, is only happening with REACT, I don't know if something has changed or how to fix it to use the 2.20 version because I couldn't find any documentation to solve it other than changing the version to 2.22.

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

**Pull request merged**